components/multiselect/multiselect_list.jsx

```diff
@@ -177,9 +177,6 @@ MultiSelectList.defaultProps = {
 MultiSelectList.propTypes = {
     options: PropTypes.arrayOf(PropTypes.object),
     optionRenderer: PropTypes.func,
-    page: PropTypes.number,
-    perPage: PropTypes.number,
```

Comment:
Here we can remove `perPage` from the `defaultProps` and stop passing `page`, `perPage` and `onPageChange` from the `multiselect.jsx` file.